.buildkite/scripts/unset-secrets.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+# Unset all variables ending with _SECRET
```

Comment:
What about adding the 5 possible suffixes here ? The default suffixes mentioned in the docs: https://buildkite.com/docs/pipelines/managing-log-output#redacted-environment-variables

For instance, in elastic-package it is used also SECRET_KEY and ACCESS_KEY
https://github.com/elastic/elastic-package/blob/f112509da6b8a8b9a4e54dfa2dec4a20fed74b23/.buildkite/hooks/pre-command#L50-L51

Reply:
I think making it so that it unsets all of them would be better.
This script would make more sense to be part of our CI image, so we don't actually need to add it. But this is something we can do afterward.